client/src/components/about/style.js

```diff
@@ -0,0 +1,56 @@
+import { createStyles } from '@material-ui/core';
+const styles = createStyles((theme) => ({
```

Comment:
please decrease white space 
and use the same font family as figma also font wieght and size
please try to make it look as figma as much as you can